
package com.barryku.cloud.sforce.partner;

import javax.xml.ws.WebFault;


/**
 * This class was generated by Apache CXF 2.4.2
 * 2011-09-11T16:54:21.654-07:00
 * Generated source version: 2.4.2
 */

@WebFault(name = "MalformedQueryFault", targetNamespace = "urn:fault.partner.soap.sforce.com")
public class MalformedQueryFault_Exception extends Exception {
    
    private com.barryku.cloud.sforce.partner.MalformedQueryFault malformedQueryFault;

    public MalformedQueryFault_Exception() {
        super();
    }
    
    public MalformedQueryFault_Exception(String message) {
        super(message);
    }
    
    public MalformedQueryFault_Exception(String message, Throwable cause) {
        super(message, cause);
    }

    public MalformedQueryFault_Exception(String message, com.barryku.cloud.sforce.partner.MalformedQueryFault malformedQueryFault) {
        super(message);
        this.malformedQueryFault = malformedQueryFault;
    }

    public MalformedQueryFault_Exception(String message, com.barryku.cloud.sforce.partner.MalformedQueryFault malformedQueryFault, Throwable cause) {
        super(message, cause);
        this.malformedQueryFault = malformedQueryFault;
    }

    public com.barryku.cloud.sforce.partner.MalformedQueryFault getFaultInfo() {
        return this.malformedQueryFault;
    }
}
